extractor/src/extract/extractor.py

```diff
@@ -147,7 +147,10 @@ def apply_filters(self, query: Query, analysis: Analysis, pk_values) -> Query:
         """ Augment the sql alchemy query with filters from the analysis.
         """
         if pk_values is not None:
-            query = query.filter(self.get_column(analysis.primary_key_column).in_(pk_values))
+            if len(pk_values) == 1:
```

Comment:
Should we check that it's a list? I never know if it should be done with untyped languages...

Reply:
no I don't think we should (it's already assumed there, _in() takes a list).
I added type hints instead

Reply:
Fine by me 👍